### PR TITLE
Fix 61

### LINKED ISSUE #61: Indice Checklist for xclim 1.0 Milestone
In reference to [this spreadsheet of ICCLIM indicators](https://docs.google.com/spreadsheets/d/1TRWfgWOmFxft-HsddJPUT-yDM48MLkgs4xQekAIqDRw/edit#gid=18087391) and building off of #1 

**For short term indicator development tracking: [Indicator Development Priorities](https://docs.google.com/spreadsheets/d/1TRWfgWOmFxft-HsddJPUT-yDM48MLkgs4xQekAIqDRw/edit#gid=539742109)**

When a function has satisfied the following parameters, please check them off. An indicator needs:

- To be coded and placed within an appropriate file (indices.py or others);
- With a clear docstring that contains parameters;
- Containing tests (pytests or unittests) that test for desired and undesired behaviour (e.g. negative degree days).

Indicators are as follows:
- [x]  CDD
- [x]  CFD
- [x]  CSDI
- [x]  CSU
- [ ]  CW
- [x]  CWD
- [x]  ETR
- [x]  FD
- [x]  GD4
- [x]  HD17
- [x]  ID
- [x]  R10mm
- [x]  R20mm
- [x]  R75p
- [x]  R75pTOT
- [x]  R95p
- [x]  R95pTOT
- [x]  R99p
- [x]  R99pTOT
- [x]  RR1
- [x]  RX1day
- [x]  RX5day
- [ ]  SD1
- [ ]  SD50cm
- [ ]  SD5cm
- [x]  SDII
- [x]  SU
- [x]  TG10p
- [x]  TG90p
- [x]  TN10p
- [x]  TN90p
- [x]  TNn
- [x]  TNx
- [x]  TR
- [x]  TX10p
- [x]  TX90p
- [x]  TXn
- [x]  TXx
- [x]  vDTR
- [ ]  WD
- [x]  WSDI
- [ ]  WW
- [ ] CD
- [x] DTR
- [x] PRCPTOT
- [ ] SD
- [x] TG
- [x] TN
- [x] TX
- [x] GSL

**Please check the boxes or remove them as progress and priorities shift for xclim 1.0.**

### PR DESCRIPTION
**Pull request by huard**
* **Please check if the PR fulfills these requirements**
- [x] This PR addresses an already opened issue (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated
- [ ] bumpversion (minor / major / patch) has been called
- [ ] Tags have been pushed

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, etc.)
- Added indices to ICCLIM module
- Added indices `days_over_precip_thresh` and `fraction_over_precip_thresh`

* **Does this PR introduce a breaking change?** (Has there been an API change?)
No

* **Other information**:

* When merging, please put `fixes #{issue number}` in your comment to auto-close the issue that your PR addresses. Thanks!
Partially fixes #61

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@huard for the autoformatting to work, do a `pip install pre-commit` or `conda install -c conda-forge pre_commit` and then run `pre-commit install` on the top level of xclim. It will be slow the first time it runs but it should do all the formatting on modified files.

**Comment by huard** [comment]
Thanks for the tip. Done.

File: xclim/indices/_threshold.py [hunk 1]
```diff
@@ -649,6 +650,46 @@ def maximum_consecutive_dry_days(pr, thresh="1 mm/day", freq="YS"):
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("days", tasmax="[temperature]", thresh="[temperature]")
+def maximum_consecutive_summer_days(tasmax, thresh="25 degC", freq="YS"):
```
Comment (tlogan2000) [inline xclim/indices/_threshold.py]: To be consistent with ```tx_days_above``` I would rename to `maximum_consecutive_tx_days_above`.   Use case could be consecutive days above 0 deg C in winter or something similar

**Comment by huard** [comment]
Ok to merge despite test coverage decrease ?

**Pull request merged**